modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/sync/SyncServer.kt

```diff
@@ -167,5 +167,8 @@ interface SyncServer {
     suspend fun getBookmarkList(@Header("Authorization") authorization: String, @Body request: BookmarkRequest): BookmarksResponse
 
     @POST("/user/podcast_rating/add")
-    suspend fun addPodcastRating(@Header("Authorization") authorization: String, @Body request: PodcastRatingAddRequest): PodcastRatingAddResponse
+    suspend fun addPodcastRating(@Header("Authorization") authorization: String, @Body request: PodcastRatingAddRequest): PodcastRatingResponse
+
+    @POST("/user/podcast_rating/show")
+    suspend fun getPodcastRating(@Header("Authorization") authorization: String, @Body request: PodcastRatingShowRequest): PodcastRatingResponse
```

Comment:
I wonder if we should do this as Protocol Buffers instead of JSON, similar to how we have with bookmarks. Maybe this could be in a future PR.

It would involve adding the following to `modules/services/protobuf/src/main/proto/sync_api.proto`

```
message PodcastRating {
    string podcast_uuid = 1;
    google.protobuf.Timestamp modified_at = 3;
    uint32 podcast_rating = 4;
}

message PodcastRatingAddRequest {
    string podcast_uuid = 1;
    uint32 podcast_rating = 2;
}

message PodcastRatingGetRequest {
    string podcast_uuid = 1;
}
```

and using the HTTP header

```
@Headers("Content-Type: application/octet-stream")
```

Reply:
Done! [685a5b0](https://github.com/Automattic/pocket-casts-android/pull/2483/commits/685a5b081baa4dc0240f7fbb774f5911bfdf5afe)